CODEOWNERS

```diff
@@ -14,6 +14,7 @@
 
 @open-telemetry/erlang-approvers
 
-/instrumentation/opentelemetry_cowboy @bryannaegele @tsloughter
+/instrumentation/opentelemetry_cowboy  @bryannaegele @tsloughter
 /instrumentation/opentelemetry_ecto    @bryannaegele @tsloughter
 /instrumentation/opentelemetry_phoenix @bryannaegele @tsloughter
+/instrumentation/opentelemetry_redix   @bryannaegele @tsloughter
```

Comment:
@bryannaegele  Aren't the submitters supposed to be codeowners instead of us?

Reply:
That's wasn't totally clear to me. Updated to my user, following the pattern I saw at #6